modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/Flamingos2.scala

```diff
@@ -85,18 +86,27 @@ object Flamingos2 {
     case _                    => WindowCover.OPEN
   }
 
+  implicit def grismFromSPDisperser(d: Disperser): Grism = d match {
```

Comment:
Do you get a warning here. It was kind of frowned upon to do this kind of direct conversion though in this case it seems fine